tests/RobotFramework/tests/tedge_agent/device_profile/device_profile_operation.robot

```diff
@@ -1,48 +1,52 @@
 *** Settings ***
 Resource            ../../../resources/common.resource
+Library             Collections
+Library             OperatingSystem
 Library             ThinEdgeIO
 Library             Cumulocity
-Library             OperatingSystem
-Library             Collections
 
-Force Tags          theme:tedge_agent
 Suite Setup         Custom Setup
 Test Setup          Custom Test Setup
 Test Teardown       Get Logs
 
-*** Test Cases ***
+Test Tags           theme:tedge_agent
 
+
+*** Test Cases ***
 Send device profile operation from Cumulocity IoT
-    ${config_url}=    Create Inventory Binary    tedge-configuration-plugin    tedge-configuration-plugin    file=${CURDIR}/tedge-configuration-plugin.toml
+    ${config_url}=    Create Inventory Binary
+    ...    tedge-configuration-plugin
+    ...    tedge-configuration-plugin
+    ...    file=${CURDIR}/tedge-configuration-plugin.toml
 
     ${PROFILE_NAME}=    Set Variable    Test Profile
     ${PROFILE_PAYLOAD}=    Catenate    SEPARATOR=\n    {
-    ...        "firmware": {
-    ...            "name":"tedge-core",
-    ...            "version":"1.0.0",
-    ...            "url":"https://abc.com/some/firmware/url"
-    ...        },
-    ...        "software":[
-    ...            {
-    ...                "name":"jq",
-    ...                "action":"install",
-    ...                "version":"latest",
-    ...                "url":""
-    ...            },
-    ...            {
-    ...                "name":"tree",
-    ...                "action":"install",
-    ...                "version":"latest",
-    ...                "url":""
-    ...            }
-    ...        ],
-    ...        "configuration":[
-    ...            {
-    ...                "name":"tedge-configuration-plugin",
-    ...                "type":"tedge-configuration-plugin",
-    ...                "url":"${config_url}"
-    ...            }
-    ...        ]
+    ...    "firmware": {
+    ...    "name":"tedge-core",
+    ...    "version":"1.0.0",
+    ...    "url":"https://abc.com/some/firmware/url"
+    ...    },
+    ...    "software":[
+    ...    {
+    ...    "name":"jq",
+    ...    "action":"install",
+    ...    "version":"latest",
+    ...    "url":""
+    ...    },
+    ...    {
+    ...    "name":"tree",
+    ...    "action":"install",
+    ...    "version":"latest",
+    ...    "url":""
+    ...    }
+    ...    ],
+    ...    "configuration":[
+    ...    {
+    ...    "name":"tedge-configuration-plugin",
+    ...    "type":"tedge-configuration-plugin",
+    ...    "url":"${config_url}"
+    ...    }
+    ...    ]
```

Comment:
Hmm... Not great, but I understand why.

Reply:
Yeah I think we can add a todo to investigate a better way to format it so it is more readable in a future PR